packages/drivers/routerlicious-driver/src/documentService.ts

```diff
@@ -196,13 +196,19 @@ export class DocumentService implements api.IDocumentService {
                         }),
                     },
                     async () => {
-                        const newOrdererToken = await this.tokenProvider.fetchOrdererToken(
-                            this.tenantId,
-                            this.documentId,
-                            refreshToken,
-                        );
-                        this.ordererRestWrapper.setToken(newOrdererToken);
-                        return newOrdererToken;
+                        try {
+                            const newOrdererToken = await this.tokenProvider.fetchOrdererToken(
+                                this.tenantId,
+                                this.documentId,
+                                refreshToken,
+                            );
+                            this.ordererRestWrapper.setToken(newOrdererToken);
+                            return newOrdererToken;
+                        }
+                        catch (err: any) {
+                            err.message = err.message.concat(": Token provider failed to fetch orderer token.")
```

Comment:
This is not good - it will not aggregate well in telemetry.
Take a look at toInstrumentedOdspTokenFetcher, and usage of wrapError() there.